### PR TITLE
Add period to Item's message constraints

### LINKED ISSUE #128: Add period to Item's message constraints
image
No full stop at the end.

### PR DESCRIPTION
**Pull request by awdse22**
Close #128

### CODE REVIEW

**Review by FooChao — APPROVED** [review]
LGTM. (I rmb my fullstop)

**Comment by kaixuan477** [comment]
> LGTM. (I rmb my fullstop)

I guess we will need to fully review all the command and class messages to standardize full stops, because I realised quite a number of them are inconsistent.

File: src/main/java/seedu/sellsavvy/model/order/Item.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 public class Item {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Item can contain any printable characters, but it should not be blank";
+            "Item can contain any printable characters, but it should not be blank.";
```
Comment (yu-sutong) [inline src/main/java/seedu/sellsavvy/model/order/Item.java:13]: Very sharp and keen observation made. Good job!

**Review by yu-sutong — APPROVED** [review]
Amazing!

**Pull request merged**